cloudfoundry/azure-cloud-foundry-service-sample/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```

Comment:
There is maven wrapper in the root folder. Is it necessary to have it for each module?